pkg/scheduler/framework/runtime/framework.go

```diff
@@ -930,7 +930,7 @@ func (f *frameworkImpl) WaitOnPermit(ctx context.Context, pod *v1.Pod) (status *
 		return nil
 	}
 	defer f.waitingPods.remove(pod.UID)
-	klog.V(4).Infof("pod %q waiting on permit", pod.Name)
+	klog.V(4).InfoS("pod waiting on permit", "pod", klog.KObj(pod))
```

Comment:
why this one in particular? what is the criteria we are following when selecting which logs to migrate?